src/common/types.ts

```diff
@@ -1,18 +1,16 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-import { DocumentSelector, NotebookDocument, Event, NotebookConcatTextDocument, Disposable } from 'vscode';
-
-export const IVSCodeNotebook = Symbol('IVSCodeNotebook');
-export interface IVSCodeNotebook {
-    readonly notebookDocuments: ReadonlyArray<NotebookDocument>;
-    readonly onDidOpenNotebookDocument: Event<NotebookDocument>;
-    readonly onDidCloseNotebookDocument: Event<NotebookDocument>;
-    createConcatTextDocument(notebook: NotebookDocument, selector?: DocumentSelector): NotebookConcatTextDocument;
-}
+import { Disposable } from 'vscode';
+import * as protocol from 'vscode-languageclient';
 
 export interface IDisposable {
     dispose(): void | undefined;
 }
 
 export type TemporaryFile = { filePath: string } & Disposable;
+
+// Type for refresh notebook to pass through LSP
+export type RefreshNotebookEvent = {

```

Comment:
This is the parameters for the new 'notebook.refresh' command that would be passed to pylance.